### PR TITLE
-Added activity for mission modes

### LINKED ISSUE #23: TLE's should be recorded upon entering & exiting a mission mode
MM1, MM2, & MM3 states should include an activity to record TLE at start and end of state

### PR DESCRIPTION
**Pull request by AstroNiashree**

### CODE REVIEW

File: fsw-architecture/operating-mode-as-fsm.puml [hunk 1]
```diff
@@ -22,14 +22,17 @@ state FiniteStateMachine {
       state MissionMode1
             MissionMode1 : Record impedence probe data to event log
             MissionMode1 : for next 2 minutes at 1 second intervals
+            MissionMode1 : Record TLE entering and existing MM1
```
Comment (capsulecorplab) [inline fsw-architecture/operating-mode-as-fsm.puml]: I would split this into lines 23 and 25, one for entering and one for exiting mission mode

File: fsw-architecture/operating-mode-as-fsm.puml [hunk 2]
```diff
@@ -22,14 +22,17 @@ state FiniteStateMachine {
...
       state MissionMode2
             MissionMode2 : Turn on current supply to DeCs burnwire
             MissionMode2 : mechanism for 1 minute
             MissionMode2 : Record DeCS strain gauge data to event
             MissionMode2 : log for next 2 minutes at 1 second intervals
+            MissionMode2 : Record TLE entering and existing MM2
```
Comment (capsulecorplab) [inline fsw-architecture/operating-mode-as-fsm.puml]: see above comment

File: fsw-architecture/operating-mode-as-fsm.puml [hunk 3]
```diff
@@ -22,14 +22,17 @@ state FiniteStateMachine {
...
       state MissionMode3
             MissionMode3 : Record multi-spectral sensor data to event
             MissionMode3 : log for next 2 minutes at 1 second intervals
+            MissionMode3 : Record TLE entering and existing MM3
```
Comment (capsulecorplab) [inline fsw-architecture/operating-mode-as-fsm.puml]: see above comment

**Review by capsulecorplab — CHANGES_REQUESTED** [review]

**Review by capsulecorplab — APPROVED** [review]
LGTM

**Review by AstroNiashree — COMMENTED** [review]
Requested changes made

**Pull request merged**